packages/components/src/styles/prose/prose.stories.ts

```diff
@@ -9,720 +9,196 @@ const { generateTemplate } = storybookTemplate('sd-prose');
 const html = String.raw;
 
 /**
- * The `sd-prose` style is used to to add  typographic defaults to any vanilla HTML you don’t control, like HTML rendered from Markdown, or pulled from a CMS.
+ * Used as a set of `prose` classes to choose from in a richtext-editor.
+ *
+ * The set of style classes enables users to create visually engaging and dynamic content by incorporating various text styles, formatting options, and image embeds.
  */
 
 export default {
   title: 'Styles/sd-prose',
+  tags: ['!dev'],
   component: 'sd-prose',
   parameters: {
     ...parameters,
-    docs: { story: { inline: false, height: 'calc(100vh - 400px)' } }
+    design: {
+      type: 'figma',
+      url: 'https://www.figma.com/design/YDktJcseQIIQbsuCpoKS4V/Component-Docs?node-id=3255-5589&t=VCN0oU7WM1KOC3Zk-4'
+    }
   },
-  args: overrideArgs({ type: 'slot', name: 'default', value: '' }),
+  args: overrideArgs({
+    type: 'slot',
+    name: 'default',
+    value:
+      '<p>Large Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nibh justo ullamcorper odio tempor molestie phasellus dui vel id. Velit in sed non orci pellentesque vivamus nunc. At non tortor, sit neque tristique. Facilisis commodo integer hendrerit tortor.</p>'
+  }),
```

Comment:
Alfer Styling options can we have:
"Use “sd-prose” modifiers for alternative appearances" @auroraVasconcelos @coraliefeil